SmartDeviceLink-iOS.xcodeproj/project.pbxproj

```diff
@@ -3815,7 +3815,7 @@
 				5D4019AF1A76EC350006B0C2 /* SDL Example.app */,
 				5D61FA1C1A84237100846EE7 /* SmartDeviceLink.framework */,
 				5D61FA261A84237100846EE7 /* SmartDeviceLinkTests.xctest */,
-				88295697207CF68800EF056C /* SDL Example.app */,
+				88295697207CF68800EF056C /* SDL Example Swift.app */,
```

Comment:
We don't want to update the swift example app name, it should be identical to the obj-c example app name: `SDL Example`.

Reply:
fixed